onnx/defs/generator/defs.cc

```diff
@@ -33,69 +33,36 @@ ONNX_OPERATOR_SET_SCHEMA(
           updateOutputShape(ctx, 0, tensor_proto);
         }));
 
-static const char* ConstantLike_ver9_doc = R"DOC(
-Generate a tensor with specific constant value. The value can be specified by the 'value' 
-attribute. The shape of the output tensor is the same as the input tensor, if the input 
-tensor is provided, or the shape provided in the 'shape' attribute (if both are provided, 
-the input tensor shape takes precendence). The data type can be specified by the 'dtype' 
-argument. If 'dtype' is not specified, then the type of input tensor is used. If input 
-tensor is also not specified, then the type defaults to 'float'.
-
-The 'dtype' argument must be one of the data types specified in the 'DataType' enum field in the
-TensorProto message and be valid as an output type.
+static const char* ConstantAsShape_ver9_doc = R"DOC(
+Generate a tensor with given value and shape.
 )DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
-    ConstantLike,
+    ConstantAsShape,
```

Comment:
Since the shape is an input to this op, may I suggest `ConstantOfShape` as possible name for this op.